internal/manifest/oci_index.go

```diff
@@ -79,6 +83,18 @@ func (index *OCI1IndexPublic) UpdateInstances(updates []ListUpdate) error {
 	return index.editInstances(editInstances)
 }
 
+func annotationsToCompressionAlgorithmNames(annotations map[string]string) []string {
+	result := make([]string, 0)
```

Comment:
```suggestion
	result := make([]string, 0, 1)
```
to actually preallocate